torch/csrc/autograd/FunctionsManual.cpp

```diff
@@ -2360,28 +2360,25 @@ Tensor slice_backward_wrapper(
 // https://j-towns.github.io/papers/svd-derivative.pdf
 //
 // This makes no assumption on the signs of sigma.
-Tensor svd_backward(const std::vector<torch::autograd::Variable> &grads, const Tensor& self,
-          bool some, bool compute_uv, const Tensor& raw_u, const Tensor& sigma, const Tensor& raw_v) {
-  TORCH_CHECK(compute_uv,
-           "svd_backward: Setting compute_uv to false in torch.svd doesn't compute singular matrices, ",
-           "and hence we cannot compute backward. Please use torch.svd(compute_uv=True)");
-
-  auto m = self.size(-2);
-  auto n = self.size(-1);
+Tensor svd_backward(const std::vector<torch::autograd::Variable> &grads,
+          bool full_matrices, const Tensor& U, const Tensor& sigma, const Tensor& Vh) {
+  auto m = U.size(-2);
+  auto n = Vh.size(-1);
   auto k = sigma.size(-1);
   auto gsigma = grads[1];
 
-  auto u = raw_u;
-  auto v = raw_v;
+  auto u = U;
+  auto v = Vh.mH();
   auto gu = grads[0];
-  auto gv = grads[2];
+  // grads[2] has the gradient wrt Vh
+  auto gv = grads[2].defined() ? grads[2].mH() : grads[2];
```

Comment:
Oh, this will break my usage of svd_backward in https://github.com/pytorch/pytorch/pull/71126.